packages/common/src/index.ts

```diff
@@ -116,6 +116,11 @@ export function hexToRGB(hex: string, alpha: number | undefined): string {
     return `rgb(${r}, ${g}, ${b})`;
 }
 
+export enum ProjectBaseType {
```

Comment:
In my next PR I'm renaming **ProjectType** to **DbtConfigType** and ** ProjectBaseType** to **ProjectType**.